packages/drivers/routerlicious-driver/src/wholeSummaryDocumentStorageService.ts

```diff
@@ -226,4 +249,77 @@ export class WholeSummaryDocumentStorageService implements IDocumentStorageServi
         });
         await Promise.all(blobCachePutPs);
     }
+
+    // Todo: surface through @fluidframework/server-services-client
```

Comment:
Will remove this after https://github.com/microsoft/FluidFramework/pull/9882 is merged.